content/tooling/api/scripts.md

```diff
@@ -185,16 +257,15 @@ curl -X GET "https://api.cloudflare.com/client/v4/accounts/9a7806061c88ada191ed0
 
 ##### URL Parameters
 
-- `account_id`: the identifier associated with your Cloudflare account. [Find your Cloudflare Account ID](/quickstart#finding-your-cloudflare-api-keys)
+- `account_id`: the identifier associated with your Cloudflare account. [Find your Cloudflare Account ID](/quickstart/#configure)
 
 ##### Query Parameters
 
 - `include_subdomain_availability`:
 
-##### Headers ([Find Your Auth Info](/quickstart#finding-your-cloudflare-api-keys))
+##### Headers ([Find Your Auth Info](/quickstart/#configure))
```

Comment:
same suggestion as above. btw i know that this header has existed already, and you're just changing the URL, so understandable if you don't want to change it in this pr. just thought i'd mention it since i see them in the diff, you are welcome to ignore